debezium-reactive/src/main/java/io/debezium/reactive/DebeziumEvent.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.reactive;
+
+import org.apache.kafka.connect.source.SourceRecord;
```

Comment:
the import is only used on the javadoc; le'ts remove it

Reply:
I have no strong feelings on that one, IMO it's fine to keep it as it makes the JavaDoc more readable.

Reply:
So coming back to this one, I don't think we should mention `SourceRecord` here at all. It's just one way of getting key and value from this event, but it shouldn't leak through (as a concept), to a consumer of JSON for instance.

Reply:
@gunnarmorling Removed